include/onnx-mlir/Runtime/OMTensor.h

```diff
@@ -278,6 +278,11 @@ int64_t omTensorGetNumElems(OMTensor *tensor);
  */
 int64_t omTensorGetOwning(OMTensor *tensor);
 
+/**
```

Comment:
Quick question, is the doc automatically updated? If not, please regenerate it to also include this new function

Reply:
I have no idea how to generate the doc. Since check-docs didn't fail, I assume this part of the code is not covered by check-docs.